Host: fix misleading information about cordoning (#687)
harvester/docs: docs/host/host.md

## Node Maintenance

> For admin users, you can click **Enable Maintenance Mode** to evict all VMs from a node automatically. This mode is useful when you want to reboot, upgrade firmware, or replace hardware components. It will leverage the `VM live migration` feature to migrate all VMs to other nodes automatically. Note that at least two active nodes are required to use this feature.

Reviewer: ```suggestion
Admin users can enable Maintenance Mode (select **⋮ > Enable Maintenance Mode**) to automatically evict all virtual machines from a node. This mode leverages the **live migration** feature to migrate the virtual machines to other nodes, which is useful when you need to reboot, upgrade firmware, or replace hardware components. At least two active nodes are required to use this feature.
```

## Cordoning a Node

> Cordoning a node marks it as unschedulable. This feature is useful when you don't want new workload to be scheduled to a node. You can make the node schedulable again by uncordoning it.

Reviewer: ```suggestion
Cordoned nodes are marked as unschedulable. Cordoning is useful when you want to prevent new workloads from being scheduled on a node. You can uncordon a node to make it schedulable again.
```
